routemaster/state_machine/visualisation.py

```diff
@@ -0,0 +1,28 @@
+"""Visualisation code for state machines."""
+
+from typing import Dict, List, Union
+
+from routemaster.config import Action, StateMachine
+
+
+def nodes_for_cytoscape(
```

Comment:
I'm not sure how much I like the naming being specific to the library we're using on the front end.

On the one hand, this looks like a specific format that wouldn't easily be used in other tools, but on the other, it feels like we shouldn't be encoding JS dependencies in Python naming like this.

What do you think?

Reply:
I originally had the massively more vague name `convert_to_network` and change to this since the output format is specific to `cytoscape`. I'm not sure whether I like that the name is coupled to the library, however since the data format is pretty much coupled anyway, I think having the name be clear is better than the alternative.